signing/secp256k1/impl/src/main/java/tech/pegasys/signers/secp256k1/multikey/SigningMetadataTomlConfigLoader.java

```diff
@@ -67,18 +62,19 @@ public SigningMetadataTomlConfigLoader(final Path rootDirectory) {
     }
   }
 
-  Collection<SigningMetadataFile> loadAvailableSigningMetadataTomlConfigs() {
-    final Collection<SigningMetadataFile> metadataConfigs = new HashSet<>();
+  ImmutableList<SigningMetadataFile> loadAvailableSigningMetadataTomlConfigs(
```

Comment:
Why return dependency on ImmutableList? You can declare `List` instead (assuming ImmutableList extends/implements List).

Reply:
true. Done.